packages/expo-modules-test-core/android/src/main/java/expo/modules/test/core/ModuleMockHolder.kt

```diff
@@ -2,10 +2,12 @@ package expo.modules.test.core
 
 import expo.modules.core.interfaces.services.EventEmitter
 import expo.modules.kotlin.AppContext
+import expo.modules.kotlin.modules.Module
 
-data class ModuleMockHolder<T>(
-  val module: T,
+data class ModuleMockHolder<TestInterfaceType, ModuleType : Module>(
+  val module: TestInterfaceType,
   val controller: ModuleController,
   val appContext: AppContext,
-  val eventEmitter: EventEmitter
+  val eventEmitter: EventEmitter,
+  val moduleSpy: ModuleType
```

Comment:
I could possibly move this to the `ModuleController` class as `ModuleHolder` already holds a reference to the module instance.